pkg/term/proxy.go

```diff
@@ -51,6 +51,10 @@ func (r *escapeProxy) Read(buf []byte) (int, error) {
 		return nr, err
 	}
 
+	if len(r.escapeKeys) == 0 {
```

Comment:
Should this be more at the top of the function? (before the `preserve = func() {` ?)

Reply:
I guess it makes more sense, I'll push the changes quickly :)